src/sql/workbench/contrib/notebook/browser/notebook.css

```diff
@@ -79,6 +79,16 @@
 	padding-left: 18px;
 	width: 16px;
 }
+
+.notebookEditor
+	.in-preview
+	.actions-container
+	.action-item
+	.codicon.icon-run-with-parameters:before {
+	padding-left: 20px;
```

Comment:
The icon looks relatively smaller than the other ones on the toolbar so I increased the size so that it's a similar size to other icons, discussed this with Hale as well.

Reply:
mmm, that's definitely a smell that something isn't right here to me. If the icons are the same why wouldn't they be the same size in the UI? (changing the size of the SVG not affecting it makes sense to me though - the size of the SVG doesn't matter at all since we scale these to specific pixel height/widths anyways)

 But if you don't have time to investigate this now then ok - I'll probably take a look once you get this merged in.

Reply:
Without the padding:
![image](https://user-images.githubusercontent.com/23587151/113207953-1c87b380-9226-11eb-87e8-7003d5d9d609.png)

With the padding:
![image](https://user-images.githubusercontent.com/23587151/113208061-39bc8200-9226-11eb-9a9f-55c3d43ea369.png)

Just for reference. Can address in a later PR after investigating further.